asset-tests/app/src/androidTest/java/org/gearvrf/tester/AssetTextureTests.java

```diff
@@ -88,38 +88,38 @@ public void jassimpEmbeddedTextures() throws TimeoutException
     @Test
     public void jassimpLightmap() throws TimeoutException
     {
-        mHandler.loadTestModel("https://raw.githubusercontent.com/gearvrf/GearVRf-Tests/master/jassimp/lightmap/lightmap_test.fbx", 13, 0, "jassimpLightmap");
+        mHandler.loadTestModel(GVRTestUtils.ASSETS_BASE_URL + "jassimp/lightmap/lightmap_test.fbx", 13, 0, "jassimpLightmap");
     }
 
     @Test
     public void jassimpCubeWrongTex() throws TimeoutException
     {
-        mHandler.loadTestModel("https://raw.githubusercontent.com/gearvrf/GearVRf-Tests/master/jassimp/cube/cube_wrongtex.fbx", 0, 1, "jassimpCubeWrongTex");
+        mHandler.loadTestModel(GVRTestUtils.ASSETS_BASE_URL + "jassimp/cube/cube_wrongtex.fbx", 0, 1, "jassimpCubeWrongTex");
     }
 
 
     @Test
     public void x3dTexcoordTest1() throws TimeoutException
     {
-        mHandler.loadTestModel("https://raw.githubusercontent.com/gearvrf/GearVRf-Tests/master/x3d/texture_coordinates/texturecoordinatetest.x3d", 4, 0, "x3dTexcoordTest1");
+        mHandler.loadTestModel(GVRTestUtils.ASSETS_BASE_URL + "x3d/texture_coordinates/texturecoordinatetest.x3d", 4, 0, "x3dTexcoordTest1");
     }
 
     @Test
     public void x3dTexcoordTest2() throws TimeoutException
     {
-        mHandler.loadTestModel("https://raw.githubusercontent.com/gearvrf/GearVRf-Tests/master/x3d/texture_coordinates/texturecoordinatetestsubset.x3d", 3, 0, "x3dTexcoordTest2");
+        mHandler.loadTestModel(GVRTestUtils.ASSETS_BASE_URL + "x3d/texture_coordinates/texturecoordinatetestsubset.x3d", 3, 0, "x3dTexcoordTest2");
     }
 
     @Test
     public void x3dTexcoordTest3() throws TimeoutException
     {
-        mHandler.loadTestModel("https://raw.githubusercontent.com/gearvrf/GearVRf-Tests/master/x3d/texture_coordinates/texturecoordinatetestsubset2.x3d", 5, 0, "x3dTexcoordTest3");
+        mHandler.loadTestModel(GVRTestUtils.ASSETS_BASE_URL + "x3d/texture_coordinates/texturecoordinatetestsubset2.x3d", 5, 0, "x3dTexcoordTest3");
     }
 
     @Test
     public void x3dTexcoordTest4() throws TimeoutException
     {
-        mHandler.loadTestModel("https://raw.githubusercontent.com/gearvrf/GearVRf-Tests/master/x3d/texture_coordinates/texturecoordinatetestsubset3.x3d", 2, 0, "x3dTexcoordTest4");
+        mHandler.loadTestModel(GVRTestUtils.ASSETS_BASE_URL + "x3d/texture_coordinates/texturecoordinatetestsubset3.x3d", 2, 0, "x3dTexcoordTest4");
     }
 
     @Test
```

Comment:
testDownloadTextureCache() uses direct github.com URL that resolves as a redirect to githubusercontent.com. Do we need to replace this URL to the base URL? I'm not sure if this test wants to test redirects/invalid URLs or not.